R/check_pop_data.R

```diff
@@ -6,31 +6,36 @@
 #' @export
 #' @examples
 #' library(dplyr)
+#' xs_data <- readr::read_rds("https://osf.io/download//n6cp3/") %>%
+#'             as_pop_data()
+#'   check_pop_data(xs_data, verbose = TRUE)
 #'
-#' xs_data <- load_pop_data("https://osf.io/download//n6cp3/") %>%
-#'   check_pop_data()
-#'
-check_pop_data <- function(pop_data,
-                           verbose = FALSE) {
+check_pop_data <- function(pop_data, verbose = FALSE) {
   if (!is.data.frame(pop_data)) {
-    cli::cli_abort(message = .pasteN(
-      "Argument `pop_data` is not a `data.frame()`.",
-      "Provide a `data.frame()` with cross-sectional serology data per antigen isotype."
-    ))
+    cli::cli_abort(
+      class = "not a data.frame",
```

Comment:
Do we communicate all types of error classes in the package?

Reply:
@chrisorwa see above; I don't think so, since they aren't typically visible to users of our package.